parasolr/query.py

```diff
@@ -158,6 +178,30 @@ def filter(self, *args, **kwargs) -> 'SolrQuerySet':
 
         return qs_copy
 
+    def facet(self, *args: str, **kwargs) -> 'SolrQuerySet':
+        """
+        Return a new SolrQuerySet with Solr faceting enabled and facet.field
+        parameter set. Does not yet support ranged faceting.
```

Comment:
I would drop the *yet* - suspect we'll want a different method to configure range facets

Reply:
Also done.